documentation/assemblies/deploying/assembly-deploy-cluster-operator.adoc

```diff
@@ -10,6 +10,8 @@ The Cluster Operator is responsible for deploying and managing Kafka clusters wi
 
 When the Cluster Operator is running, it starts to watch for updates of Kafka resources.
 
+By default, a single Cluster Operator is deployed. You can add replicas with leader election so that additional Cluster Operators are on standby in case of disruption. For more information, see link:{BookURLUsing}#assembly-using-multiple-cluster-operator-replicas-str[Running multiple Cluster Operator replicas with leader election]. 
```

Comment:
```suggestion
By default, a single replica of the Cluster Operator is deployed. You can add replicas with leader election so that additional Cluster Operators are on standby in case of disruption. For more information, see link:{BookURLUsing}#assembly-using-multiple-cluster-operator-replicas-str[Running multiple Cluster Operator replicas with leader election]. 
```